modules/admanBidAdapter.js

```diff
@@ -0,0 +1,79 @@
+import {registerBidder} from '../src/adapters/bidderFactory';
+import * as utils from '../src/utils';
+
+const BIDDER_CODE = 'adman';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: ['video', 'banner'],
+  isBidRequestValid: function(bid) {
+    let isValid = false;
```

Comment:
The `isValid` logic can be simplified if you use `utils.deepAccess` to something like this:
```
isBidRequestValid: function(bid) {
  const isValid = _validateId(utils.deepAccess(bid, 'params.id'));
  if (!isValid) {
    utils.logError('Adman id parameter is required. Bid aborted.');
  }
  return isValid;
}
```